packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -79,8 +80,14 @@ export function makeScreenshotEditorComponent({ h, imageBuffer, dialog }: Factor
       const cropper = croppingRef.current;
       const imageDimensions = constructRect(getContainedSize(imageBuffer));
       if (cropper) {
-        cropper.width = imageDimensions.width;
-        cropper.height = imageDimensions.height;
+        cropper.width = imageDimensions.width * DPI;
+        cropper.height = imageDimensions.height * DPI;
+        cropper.style.width = `${imageDimensions.width}px`;
+        cropper.style.height = `${imageDimensions.height}px`;
+        const ctx = cropper.getContext('2d');
+        if (ctx) {
+          ctx.scale(DPI, DPI);
+        }
```

Comment:
This reduces the crop rectangle blurriness, which is a known issue on retina screens: https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio